internal/generator/backend/template/representations/solution.go.tmpl

```diff
@@ -139,18 +139,15 @@ func (s *solution) solveWithHint(vID int, h *compiled.Hint) error {
 	// is the output of another hint. 
 	// it is safe to recursively solve this with the parallel solver, since all hints-output wires 
 	// that we can solve this way are marked to be solved with the current constraint we are processing.
-	solveOrPanic := func(t compiled.Term) {
+	solveOrPanic := func(t compiled.Term) error {
```

Comment:
semantically, maybe we shouldn't call it solveOrPanic then :)

Reply:
:), yup.